irma-css/src/dist.scss

```diff
@@ -1,3 +1,2 @@
-
-@charset 'utf-8';
+@charset "utf-8";
```

Comment:
According to the specs, the `@charset` rule should always be double quoted
https://developer.mozilla.org/en-US/docs/Web/CSS/@charset